Dockerfile

```
@@ -1,147 +1,118 @@
+# Copy the rest of the renderer code and build frontend assets
+COPY renderer/ ./
+RUN npm run build && npm run postbuild
+
+# ----------------------------------------
+# 2. DEPS STAGE
 # ----------------------------------------
 FROM base AS deps
 
-# Install CA certificates first to fix SSL issues
+# Update and install dependencies
 RUN apt-get update -y && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)